apps/server/lib/lexical/server/project/intelligence.ex

```diff
@@ -51,6 +51,35 @@ defmodule Lexical.Server.Project.Intelligence do
     defp prefixes_match?(_, _, _) do
       false
     end
+
+    require Logger
+
+    defp discover_existing_structs(%__MODULE__{} = state) do
+      # This might be a performance / memory issue on larger projects. It
+      # iterates through all modules, loading each as necessary and then removing them
+      # if they're not already loaded to try and claw back some memory
+
+      for {module_name_charlist, _, loaded?} <- :code.all_available(),
```

Comment:
is `:code.all_available()` returning all the modules in the `:server` app?

Reply:
Yes

Reply:
i see, we should specifically exclude the server app from this, correct?